sites/public/src/components/listings/search/ListingsSearch.module.scss

```diff
@@ -2,26 +2,30 @@
   // This is a not-ideal way to do "fill window minus header and filter modal" however
```

Comment:
These styles manage changes to the filter bar that is now full width + has new buttons on mobile